openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderTrainTest.java

```diff
@@ -91,15 +91,16 @@ private Dataset createDataset(final DatasetSchema schema) {
      *
      * @param schema The schema that the dataset will comply to.
      * @param size   The number of instances of the dataset.
+     *
```

Comment:
I don't think it's usual to add an empty line before the @ return. once again, it's a nitpick